### PR TITLE
Fix page return handling in config edit and summary, reset active pag…

### PR DESCRIPTION
**Pull request by Filip-A25**
…e on ConfigDisplay first render

### CODE REVIEW

File: src/modules/configurator/components/ConfigOptionsButton.tsx [hunk 1]
```diff
@@ -8,7 +8,7 @@ export function ConfigOptionsButton({
   setIsOptionsDropdownOpen,
 }: Props) {
   return (
-    <button onClick={() => setIsOptionsDropdownOpen(!isOptionsDropdownOpen)}>
+    <button onClick={(e) => setIsOptionsDropdownOpen(!isOptionsDropdownOpen)}>
```
Comment (StjepanDzojic) [inline src/modules/configurator/components/ConfigOptionsButton.tsx]: You dont need the event here

File: src/modules/configurator/components/ConfigNavbar.tsx [hunk 2]
```diff
@@ -6,24 +6,24 @@ import { PaginationDisplay } from "./PaginationDisplay";
 import { ConfigOptionsDisplay } from "./ConfigOptionsDisplay";
 
 interface Props {
-  returnPath: string;
+  returnOnClick: () => void;
```
Comment (mrkaza) [inline src/modules/configurator/components/ConfigNavbar.tsx]: Could also be of type `VoidFunction`

File: src/modules/configurator/components/ConfigOptionsButton.tsx (hunk 1 above)
Comment (mrkaza) [inline src/modules/configurator/components/ConfigOptionsButton.tsx]: Why did you put the `e` here when you are not using it?

**Review by StjepanDzojic — APPROVED** [review]

File: src/modules/configurator/components/ConfigNavbar.tsx (hunk 2 above)
Comment (Filip-A25) [reply src/modules/configurator/components/ConfigNavbar.tsx]: Done

File: src/modules/configurator/components/ConfigOptionsButton.tsx (hunk 1 above)
Comment (Filip-A25) [reply src/modules/configurator/components/ConfigOptionsButton.tsx]: Removed, stayed there by mistake.

Comment (Filip-A25) [reply src/modules/configurator/components/ConfigOptionsButton.tsx]: Removed, stayed there by mistake.

**Pull request merged**